theme/typography.ts

```diff
@@ -28,6 +28,12 @@ const textStyles = typographyTokensJson[
   "text-styles"
 ] as unknown as TypoToken[];
 
+type TextPreset = Extract<
+  keyof (typeof typographyTokensJson)["text-styles"],
+  string | number
```

Comment:
Using `Extract` is a good idea, but the `text-styles` object doesn't have `number` keys, only `string`.